bioimageio/core/prediction_pipeline/_prediction_pipeline.py

```diff
@@ -144,26 +91,6 @@ def input_specs(self):
     def output_specs(self):
         return self._output_specs
 
-    @property
```

Comment:
so, with these properties gone, are they supposed to be queried from the model?

Reply:
they are supposed to be accessed through `input_specs` and `output_specs` (which are set from `model.inputs` and `model.outputs` respectively. This avoids redefining everything as the (non-raw/derived) nodes are now part of this core module.

Reply:
good to know. Sounds reasonable :)